.github/workflows/flatpak-devel.yml

```diff
@@ -23,8 +23,10 @@ jobs:
       env:
         REPO: https://github.com/${{ github.repository }}
         REF: ${{ github.ref }}
+    - name: Add SHORT_SHA environment variable
+      run: echo "SHORT_SHA=`echo ${GITHUB_SHA::7}`" >> $GITHUB_ENV
```

Comment:
Why do you write into a file? Shouldn't  something like the following work?
```bash
export SHORT_SHA=`echo ${GITHUB_SHA::7}`
```

Reply:
No, with github actions the env variables set with `export` do not stick to the next step. Adding them the `$GITHUB_ENV` is the way to make them available later.
An alternative to this syntax would be to use `set-output` functionality like we do [here](https://github.com/jacktrip/jacktrip/blob/dev/.github/workflows/jacktrip.yml#L99) but I think what I proposed is a bit shorter (we don't need to specify step id). 
However, if you prefer, we could do `set-output`, it's not that different.

Reply:
Interesting.

Reply:
Any preference on the current approach vs `set-output`?

Reply:
I prefer the current approach.